compiler/rustc_resolve/src/build_reduced_graph.rs

```diff
@@ -118,6 +119,13 @@ impl<'ra, 'tcx> Resolver<'ra, 'tcx> {
             return module.copied();
         }
 
+        //if def_id.is_crate_root() && !def_id.is_local() {
```

Comment:
Expected this to hold, but it fails when compiling libc. Likely here: https://github.com/rust-lang/libc/blob/e8b01525c30545041af1ad7ba24c05ee3251016b/src/lib.rs#L33-L37